### ISSUE TITLE
How to distribute the data to ep.train.txt, ep.dev.txt, and ep.test.txt? what's the purpose of these files?

### ISSUE THREAD

**Issue by wltz** [ottokart/punctuator2#38]
head -n -400000 step2.txt > ./out/ep.train.txt
tail -n 400000 step2.txt > step3.txt
head -n -200000 step3.txt > ./out/ep.dev.txt
tail -n 200000 step3.txt > ./out/ep.test.txt

Hi ottokart, 
Could you elaborate on how to distribute the data from corpus to these three files? And what's the purpose  of these files?
I have a small corpus file, 65k lines and about 3M words. So, I need to know how should I distribute the data to these files. Thanks!

**Comment by Artificial2idiot** [comment]
i think the author just divide this corpus into two parts, the first part used as train set is about 1600000 lines ,and the second part is about 400000 lines. Then the second part is divided into two sets with the same capacity, and these two sets are utilized as validation set and test set respectively.

**Comment by abdallahkotb5566** [comment]
Where can I find dataset ? 
and code sys.arg[0] make error in all files
